src/parser/view_transform.c

```diff
@@ -3779,16 +3779,17 @@ pt_copypush_terms (PARSER_CONTEXT * parser, PT_NODE * spec, PT_NODE * query, PT_
       /* copy terms */
       query->info.dblink_table.pushed_pred = parser_copy_tree_list (parser, term_list);
```

Comment:
how about copying the exact node (inside of the CAST expression node) instead of defining a custom print.

Reply:
The tree traverse and copy exact node seems complexed some. So, I keep the code as is.

Reply:
Ok, I will try it.